src/BrefKernel.php

```diff
@@ -49,33 +49,43 @@ public function boot()
         return parent::boot();
     }
 
-    protected function prepareCacheDir(string $staticCacheDir, string $tempCacheDir): void
+    /**
+     * Return the pre-warmed directories in var/cache/[env] that should be copied to
+     * a writable directory in the Lambda environment.
+     */
+    protected function getWritableCacheDirectories(): array
+    {
+        return [];
```

Comment:
The pools folder must be writable for multiple reasons.

- For example the [`symfony/property-info`](https://symfony.com/doc/current/components/property_info.html) will generate cache entries on runtime. Those write attemts fail silently in prod mode if you don't configure the logging to output warnings.
- All app caches will not work when the pools folder is not writable. I'd like if the the simplest cache works out of the box without having to directly configure an external caching service.

Now it might be possible to change the default `app.cache` folder and recommend silencing failing `system.cache` writes... or change that implementation as well.
https://symfony.com/doc/4.4/cache.html#configuring-cache-with-frameworkbundle

Reply:
If you warmed up the cache and not using the "filesystem cache pool", then you are safe. This is what you should do if you are serious about performance. 

But I get your point about the pools folder if you havn't warmed the cache. I think we should solve that with documentation. 

We should decide if we want: 
- Users to change config if they prewarm
- Users to change config if they are lazy

Hm... Lazy people will not change anything... You are correct. I should change this. :/

Reply:
Even if you warm your cache correctly, there are still writes to the cache during runtime. Or at least in symfony 4.4 there are though the mentioned property-info component.

Reply:
I have to investigate if that is correct. I was sure you could deploy symfony 4.4 on a read only filesystem.

Reply:
You can (as i accidentally noticed in #16) but it isn't ideal, i think.
I configured logging like this:
```
main:
    type: stream
    path: "%env(resolve:LOG_FILE)%"
    level: debug # warning would be enough
```
And you'll notice failed write attempts. Now we could say this is a bug or a missing cache warmer in that component.

If we assume it is a bug in that component (and it is the only component), then we just have the `app.cache` to worry about. For that we could try to change the default since we have access to the kernel.
For that we can